### ISSUE TITLE
plot_ic_ts seems not work well if len(ic.columns) == 1

### ISSUE THREAD

**Issue by zilongli** [quantopian/alphalens#115]
In the function plot_ic_ts, if ic only has one column, then
 
```
num_plots = len(ic.columns)
if ax is None:
    f, ax = plt.subplots(num_plots, 1, figsize=(18, num_plots * 7))
    ax = ax.flatten()
```
will raise an error of 
AttributeError: 'AxesSubplot' object has no attribute 'flatten'

**Comment by zilongli** [comment]
a simple way is to change line
`ax = ax.flatten()`
into
`ax = np.array([ax]).flatten()`

**Comment by twiecki** [comment]
Thanks, @zilongli. Want to submit a PR with the fix? Should probably use `np.asarray()`.

**Comment by luca-s** [comment]
@zilongli I will go on and fix this issue. Please let me know if you are planning to do it instead.

**Comment by luca-s** [comment]
Fixed in #126
